snuba/datasets/cdc/groupedmessage.py

```diff
@@ -0,0 +1,58 @@
+from snuba.clickhouse import ColumnSet, DateTime, Nullable, String, UInt
+from snuba.datasets import DataSet
+from snuba.datasets.cdc.groupedmessage_processor import GroupedMessageProcessor
+from snuba.datasets.schema import ReplacingMergeTreeSchema
+
+
+class GroupedMessageDataSet(DataSet):
+    """
+    This is a full clone of postgres groupedmessage
+    generated through change capture.
+    """
+
+    def __init__(self):
+        columns = ColumnSet([
```

Comment:
@JTCunning, these are the columns actively used today in search backend to query groups. The commented ones are those we can skip.